spec/controllers/container_image_registry_controller_spec.rb

```diff
@@ -18,7 +18,7 @@
     get :show, :id => container_image_registry.id
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
-    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registry",
+    expect(assigns(:breadcrumbs)).to eq([{:name => "Container Image Registries",
```

Comment:
> @dkorn this doesn't seem related.

@simon3z while making this refactor I noticed there was a small bug in the Image Registries  ```display_name``` method: "Container Image Registry" should have been "Container Image Registries" to maintain consistency with all other entities.
The refactor handled the fix, but the spec needed a change as well.